README.md

```diff
@@ -120,17 +139,33 @@ Methods:
 | stroke linecap  | std::string | null             |
 | stroke linejoin | std::string | null             |
 
-| Method            | Parameter type | Description           | 
+| Method            | Parameter type | Description           |
 |-------------------|----------------|-----------------------|
 | SetStrokeColor    | svg::Point     | Sets stroke color.    |
 | SetFillColor      | svg::Point     | Sets fill color.      |
 | SetStrokeWidth    | float          | Sets stroke width.    |
 | SetStrokeLinecap  | std::string    | Sets stroke linecap.  |
 | SetStrokeLineJoin | std::string    | Sets stroke linejoin. |
 
+### svg::Section
+
+Section is an object type with unique properties. It doesn't have most properties, listed above and doesn't specify
+any figure itself, instead section may be considered as the set of different objects, such as Circle, Text etc.
+Section has the only one method `Render` which renders all contained figures in their order.
+
+How to use svg::Section:
+
+1. Create an object of type svg::SectionBuilder.
+2. Use the method `Add` of the builder to add all figures in a set.
+3. Use method `Build` of builder to obtain a `svg::Section` object.
+
+The main purpose of creating Section type is the possibility of copying a set of objects without copying objects
```

Comment:
The purpose of Section is chip copying of a set of objects without copying the objects themselves.

Reply:
Done.